sw/airborne/modules/relative_localization_filter/relative_localization_filter.c

```diff
@@ -37,40 +37,76 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#include "discrete_ekf.h"
 
 #ifndef RL_NUAVS
 #define RL_NUAVS 4 // Maximum expected number of other UAVs
 #endif
 
+/*
+ * NO_NORTH = 1 : The filter runs without a heading reference.
+ * NO_NORTH = 0 : The filter runs while using a shared reference heading.
+ */
+#ifndef NO_NORTH
```

Comment:
This is not a very descriptive setting name, can you add something with the module name, recommendation RELATIVE_LOCALIZATION_NO_NORTH or RELATIVE_LOCALIZATION_FILTER_NO_NORTH or RL_NO_NORHT like the other settings.